freedomofpress/securedrop
# Update cryptography requirement in admin, cffi everywhere

## Status

Ready for review

## Description of Changes

Update admin cryptography requirement to 3.4.7. Update cffi to 1.14.5 globally.

Fixes #5978.

## Testing

On an admin workstation:
- check out this branch with `git checkout update-admin-cryptography`
- update the admin virtualenv with `securedrop-admin setup`
- `admin/.venv3/bin/pip show cryptography cffi` should show cryptography at 3.4.7 and cffi at 1.14.5
- (re)move `install_files/ansible-base/tor_v3_keys.json` and regenerate with `securedrop-admin generate_v3_keys`
- check that the new `install_files /ansible-base/tor_v3_keys.json` was generated without error
- for bonus points, run `securedrop-admin install/tailsconfig/verify`

## Deployment

Admin workstations will need to be updated with `securedrop-admin setup`.

## Checklist

### If you made changes to `securedrop-admin`:


### If you made non-trivial code changes:


Choose one of the following:


### If you added or updated a code dependency:

Choose one of the following:

- [x] I have performed a diff review (for CFFI; cryptography 3.4.7 was already reviewed) and pasted the contents to [the packaging wiki](https://github.com/freedomofpress/securedrop-debian-packaging/wiki/cffi-1.14.2-to-1.14.5)
